src/content/en/updates/2019/05/desynchronized.md

```diff
@@ -0,0 +1,193 @@
+latency issues because a web page has to synchronize graphics updates with the
+DOM. In any drawing application, latencies longer than 50 milliseconds can
+interfere with a user's hand-eye coordination, making applications difficult to
+use.   
+
+The `desynchronized` hint for `canvas.getContext()` invokes a different code
+path that bypasses the [usual DOM update
+mechanism](https://docs.google.com/presentation/d/1boPxbgNrTU0ddsc144rcXayGA_WF53k96imRH8Mp34Y/edit#slide=id.p).
+Instead the hint tells the underlying system to skip as much compositing as it
+is able and in some cases, the canvas's underlying buffer is sent directly to
+the screen's display controller. This eliminates the latency that would be
+caused by using the renderer compositor queue.
+
+## How good is it?
+
+<img src="/web/updates/images/2019/05/sintel.png" class="sintel attempt-right" alt="Simultaneous rendering of Sintel">
+
+<style>
+.sintel {
+  height: 300px;
+  width: auto;
+}
+</style>
+
+To see it in action, you need a device with a touch screen, and preferably a
+stylus. (Fingers work too.) If you have one, try the
+[2d](https://www.google.com/url?q=https://codepen.io/miguelao/full/ZjJNNw&sa=D&ust=1556721118370000&usg=AFQjCNGjpffZOOmf99D_ixBGNlYHGLiF7w)
+or
+[webgl](https://www.google.com/url?q=https://codepen.io/miguelao/full/WKZaqd&sa=D&ust=1556721118370000&usg=AFQjCNGcfmYlh3Serjw0d8o4isSYv8eywg)
+samples. For the rest of you check out this [demo by Miguel
+Casas](https://codepen.io/miguelao/full/mLLKLg), one of the engineers who
+implemented this feature. Open the demo, press play, then move the slider back
+and forth randomly and quickly. 
+
+This example uses a one-minute, twenty-one second clip from the short film
+[Sintel](https://durian.blender.org/download/) by Durian, the Blender open movie
+project. In this example, the movie is played in a `<video>` element whose
+contents are simultaneously rendered to a `<canvas>` element.  Many devices can
+do this without tearing, though devices with front buffer rendering such as
+Chrome OS for example may have tearing. (The movie is great, but heartbreaking.
+I was useless for an hour after I saw it. Consider yourself warned.)
+
+
+
+## Using the hint
+
+There's more to using low latency than adding `desynchronized` to 
+`canvas.getContext()`. I'll go over the issues one at a time.
+
+### Feature detection 
+
+Feature detection is mixed up with how you create a desynchronized canvas, so
+I'll discuss them together. First, call `canvas.getContext()` and pass it the
+new `desynchronized` hint with a value of `true`. 
+
+```javascript
+const canvas = document.querySelector('myCanvas');
+const ctx = canvas.getContext('2d', { 
+  desynchronized: true,
+  // Other options. See below.
+});
+```
+
+Next, call `getContextAttributes()` to determine if the hint is supported. Alas,
+this does not guarantee a desynchronized canvas because the feature requires
```

Comment:
If the flag is true, the canvas will be desynchronized. It wouldn't make sense to say "yeah, we're desynchronized' and still not being desynchronized, right?

Having front buffer rendering or not, is a different story, an optimization if you want, that as a side effect might cause tearing. Front buffer rendering needs to have a desynchronized canvas to work, and ATM is only implemented on ChromeOS.